presto-main/src/main/java/com/facebook/presto/security/AccessControlManager.java

```diff
@@ -121,12 +122,12 @@ public void removeCatalogAccessControl(ConnectorId connectorId)
     public void loadSystemAccessControl()
             throws Exception
     {
-        if (ACCESS_CONTROL_CONFIGURATION.exists()) {
-            Map<String, String> properties = new HashMap<>(loadProperties(ACCESS_CONTROL_CONFIGURATION));
+        if (configurationFile.exists()) {
```

Comment:
I think we should throw an exception when configuration file is explicitly specified, but it doesn't exists.